.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 jobs:
   test-arm:
     machine:
-      image: ubuntu-2004:2022.04.1
+      image: ubuntu-2004:2024.01.1

```

Comment:
The point of using 22.04 is that it is an LTS release, so we wouldn't really want to change to a quarterly release therefrom. Is there still an LTS image to use?

Reply:
Could you just specify `default` here? That should point to the LTS by default: https://discuss.circleci.com/t/linux-image-deprecations-and-eol-for-2024/50177